frontend/src/store/userStore.ts

```diff
@@ -2,29 +2,33 @@ import { ref } from 'vue';
 import { defineStore, storeToRefs } from 'pinia';
 import type { User } from '@/interfaces';
 import { userService } from '@/services';
-import { useAuthStore } from '@/store';
+import { useAuthStore, useConfigStore } from '@/store';
 
 export const useUserStore = defineStore('user', () => {
   const { getIdentityId } = useAuthStore();
   const { getKeycloak } = storeToRefs(useAuthStore());
+  const { config } = storeToRefs(useConfigStore());
 
   const userSearch = ref([] as User[]);
   const idps = ref([] as Object[]);
   const loading = ref(false);
   const userId = ref('');
+  const elevatedRights = ref(false);
 
   async function init() {
-    await getUserId();
+    await getUser();
   }
 
-  // Hydrates the logged in users ID from the COMS database
-  async function getUserId() {
+  // Hydrates the logged in users info from the COMS database
+  async function getUser() {
     if (!userId.value && getKeycloak.value.authenticated) {
       if (getIdentityId()) {
         await searchUsers({ identityId: getIdentityId() });
 
         if (userSearch.value.length) {
           userId.value = userSearch.value[0].userId;
+          elevatedRights.value = config.value.idpList.find((idp: any) =>
+            idp.idp === userSearch.value[0].idp)?.elevatedRights ?? false;
```

Comment:
Style: This is written in one-line notation, but since it's long enough it needs to spill over into another line, the find callback function should be written with a body wrapper instead. While we generally love to abuse one-liners wherever we can, we make sure that if it has to spill to a different line due to length that it uses the full curly brace body syntax instead, as it's very easy when this is revisited and edited to cause typobombs/bugs with single liners that appear across multiple lines.